packages/api/src/auth/authHeaders.ts

```diff
@@ -83,6 +103,7 @@ export const decodeAuthToken = async ({
       decoded = {
         type,
         token,
+        authToken: null,
```

Comment:
I just realized that the `authToken` isn't necessary here. It was a remnant of a prior implementation where decoded as wasn't just the decoded JWT but an object with the type, access token and then decoded token. 

I'll remove.